docker/px4-dev/Dockerfile_ros

```diff
@@ -14,6 +14,10 @@ RUN apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C3
 		ros-kinetic-mavros \
 		ros-kinetic-mavros-extras \
 		ros-kinetic-ros-base \
+		ros-kinetic-mavlink \	# MAVROS deps
+		libgeographic-dev \
+		geographiclib-tools \
+	&& geographiclib-get-geoids egm96-5
```

Comment:
Missing \ at end of the line.

Super nitpicky, but while you're at it can you also change it to one package per line and match the indenting. Thanks!

Reply:
`&& geographiclib-get-geoids egm96-5 \` is actually a script, and not part of the above `apt-get install`